### ISSUE TITLE
FR: Support for MicroOS

### ISSUE THREAD

**Issue by deandre** [tailscale/tailscale#10783]
### What are you trying to do?

OS=other-linux
VERSION=
PACKAGETYPE=
UNAME=Linux kube 6.6.9-1-default #1 SMP PREEMPT_DYNAMIC Tue Jan 2 07:19:30 UTC 2024 (61d1d44) x86_64 x86_64 x86_64 GNU/Linux

NAME="openSUSE MicroOS"
# VERSION="20240105"
ID="opensuse-microos"
ID_LIKE="suse opensuse opensuse-tumbleweed"
VERSION_ID="20240105"
PRETTY_NAME="openSUSE MicroOS"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:opensuse:microos:20240105"
BUG_REPORT_URL="https://bugzilla.opensuse.org"
SUPPORT_URL="https://bugs.opensuse.org"
HOME_URL="https://www.opensuse.org/"
DOCUMENTATION_URL="https://en.opensuse.org/Portal:MicroOS"
LOGO="distributor-logo-MicroOS"

### How should we solve this?

NA

### What is the impact of not solving this?

NA

### Anything else?

NA

**Comment by profetik-777** [comment]
@deandre I don't think we should have tailscale keep adding all kinds of versions and variants to support  - if we have sane workarounds that are not too off from officially supported methods.  There are 2 ways to do it from my understanding  - here is one. 

https://forum.tailscale.com/t/instructions-for-opensuse-microos/3634
